google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/gcs/GcsBucketContentEditorPanelTest.java

```diff
@@ -47,23 +47,32 @@
 
   @Mock private Blob directoryBlob;
   @Mock private Blob binaryBlob;
+  @Mock private Blob nestedBlob;
 
-  private BiMap<Integer, String> indexToColName =
+  private static final String DIR_NAME = "my_directory";
```

Comment:
nit: Can you move these static variables above the instance variables to maintain a logical ordering?

Reply:
done